dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -33,44 +32,44 @@ def request
   end
 
   test "non-custom level displays localized instruction after locale switch" do
-    DEFAULT_LOCALE = 'en-us'
```

Comment:
constants are class scope so making this a constant causes constant redefinition warnings.